base/system/autochk/autochk.c

```diff
@@ -5,7 +5,7 @@
  * PURPOSE:         Filesystem checker
  * PROGRAMMERS:     Aleksey Bragin
  *                  Eric Kohl
- *                  Herv� Poussineau
+ *                  Hervé Poussineau
```

Comment:
Your editor seems to have changed the encoding. In general this is undesirable, so please double-check your editor's setting and disable automatic conversion.
In this particular instance, it's a change from ANSI to UTF-8, which is a change for the better, so you can keep it.

Reply:
I used the editor on Github itself.